internal/workload/manager.go

```diff
@@ -195,6 +211,19 @@ func (w *WorkloadManager) Update(configuration models.DeviceConfigurationMessage
 	return errors
 }
 
+func (w *WorkloadManager) manageAuthFile(authFilePath, authFile string) (string, error) {
```

Comment:
I'll add a comment to this function, mainly why we need to delete the authFilePath